server/api/build.go

```diff
@@ -453,18 +453,38 @@ func PostBuild(c *gin.Context) {
 		}
 	}
 
-	// fetch the pipeline config from database
+	var yamls []*remote.FileMeta
```

Comment:
What do you think about removing the term `yaml` here. (I know it was there before, so if you don't want to change the whole code here we can also do it in a separate PR)

Reply:
-> F2 Rename Symbol :-)

Will do

Reply:
changed